packages/core/src/types/IGeometry.ts

```diff
@@ -0,0 +1,12 @@
+import { ContourSet } from '../cache/classes/ContourSet';
+import { GeometryType } from '../enums';
+
+// interface IGeometry can be array of IContourSet
+interface IGeometry {
+  id: string;
+  type: GeometryType;
+  data: ContourSet; // | Array<IClosedSurface> , etc
```

Comment:
Typically unless commented code is likely to come back "soon" I do not include it. Here do we really need to keep the comment?